gradle.properties

```diff
@@ -82,3 +82,4 @@ org.gradle.parallel = true
 
 org.gradle.internal.http.socketTimeout=120000
 org.gradle.internal.http.connectionTimeout=120000
+org.gradle.workers.max = 3
```

Comment:
Really not sure about this change. I think it will slow down builds a ton.

Reply:
yeah it will.. but also kills local machines if it uses every core for this... I'd proven it out on a 24 core box, with less "max workers" it actually worked faster than max cores.
BUT, that said, this is local changes and can be left off PR.